mods/d2k/rules/player.yaml

```diff
@@ -119,9 +119,13 @@ Player:
 		CashTickDownNotification: CashTickDown
 	DeveloperMode:
 		CheckboxDisplayOrder: 8
-	BaseAttackNotifier:
+	DamageNotifier:
+		ValidTargets: Structure
 		TextNotification: notification-base-under-attack
 		AllyTextNotification: notification-ally-under-attack
+	DamageNotifier@RadarPing:
+		InvalidTargets: Structure
+		NotifyInterval: 10000
```

Comment:
Would it make sense to have a separate `RadarPingInterval` to avoid duplicating instances?

Reply:
The second instance is based on the other PR, I copied the `@RadarPing` suffix. The second instance is for units under attack Perhaps a better instance name such as `@units` . Thinking about it, it might be better to move that to another PR.